tests/example-empty-file/expected_results.json

```diff
@@ -1,5 +1,11 @@
 {
-  "version": 1,
-  "status": "fail",
-  "message": "\u001b[31mF\u001b[0m\n\nFailures:\n\n  1) Bob encountered a declaration exception\n   Message:\n     \u001b[31mTypeError: Bob is not a constructor\u001b[0m\n   Stacktrace:\n     TypeError: Bob is not a constructor\n    at jasmine.Suite.<anonymous> (/solution/example-empty-file.spec.coffee:3:9)\n    at Object.<anonymous> (/solution/example-empty-file.spec.coffee:2:1)\n    at Object.<anonymous> (/solution/example-empty-file.spec.coffee:1:1)\n    at Module._compile (internal/modules/cjs/loader.js:999:30)\n    at Object.loadFile (/opt/test-runner/node_modules/jasmine-node/node_modules/coffeescript/lib/coffee-script/register.js:16:19)\n    at Module.load (/opt/test-runner/node_modules/jasmine-node/node_modules/coffeescript/lib/coffee-script/register.js:45:36)\n    at Function.Module._load (internal/modules/cjs/loader.js:708:14)\n    at Module.require (internal/modules/cjs/loader.js:887:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.jasmine.executeSpecsInFolder (/opt/test-runner/node_modules/jasmine-node/lib/jasmine-node/index.js:160:9)\n    at Object.<anonymous> (/opt/test-runner/node_modules/jasmine-node/lib/jasmine-node/cli.js:247:9)\n    at Module._compile (internal/modules/cjs/loader.js:999:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)\n    at Module.load (internal/modules/cjs/loader.js:863:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:708:14)\n    at Module.require (internal/modules/cjs/loader.js:887:19)\n    at require (internal/modules/cjs/helpers.js:74:18)\n    at Object.<anonymous> (/opt/test-runner/node_modules/jasmine-node/bin/jasmine-node:6:1)\n    at Module._compile (internal/modules/cjs/loader.js:999:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)\n    at Module.load (internal/modules/cjs/loader.js:863:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:708:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)\n    at /usr/local/lib/node_modules/npm/node_modules/libnpx/index.js:268:14\n\n\n\n\u001b[31m1 test, 1 assertion, 1 failure, 0 skipped\n\u001b[0m"
-}
+	"version": 2,
+	"status": "fail",
+	"tests": [
+		{
+			"name": "encountered a declaration exception",
+			"status": "fail",
+			"message": "TypeError: Bob is not a constructor"
+		}
```

Comment:
This seems not to be the actual test but the error.

Reply:
Yeah, it is because the error doesn't occur in a test case but actually outside the test case

Reply:
Yes. Can you convert this to a top-level `error` with no tests?